pkg/controllers/monitoring-stack/controller.go

```diff
@@ -46,13 +42,10 @@ import (
 
 	"github.com/go-logr/logr"
 	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
-	goctrl "github.com/rhobs/monitoring-stack-operator/pkg/controllers/grafana-operator"
 )
 
 const (
-	grafanaDatasourceOwnerName      = "monitoring-stack-operator/owner-name"
-	grafanaDatasourceOwnerNamespace = "monitoring-stack-operator/owner-namespace"
-	finalizerName                   = "monitoring-stack-grafana-ds/finalizer"
+	finalizerName = "monitoring-stack-grafana-ds/finalizer"
```

Comment:
we shouldn't need the finalizer that was set to keep the stack object until grafana-datasource is deleted.

Reply:
Removed finalizer as currently would not be used for anything.